internal/dcache/rpc/server/handler.go

```diff
@@ -1767,7 +1767,7 @@ func writeChunkAndHash(chunkPath, hashPath *string, data *[]byte, hash *string)
 	//
 	// Write the chunk using buffered IO mode if,
 	//   - Write IO type is configured as BufferedIO, or
-	//   - The requested offset and length is not aligned to file system block size.
+	//   - The write length (or chunk size) is not aligned to file system block size.
```

Comment:
this is not correct, it's indeed offset and length that's checked for alignment.